frontend/src/containers/AdminPanel/AdminPanel.js

```diff
@@ -68,6 +70,11 @@ class AdminPanel extends Component {
   render() {
     const { user, setJwt } = this.props;
     const { activeTab, recipes, users } = this.state;
+    const isAdmin = isNil(user) ? 'loadingPage' : user.is_admin;
+
+    if (!localStorage.jwt) return <Redirect to="/" />;
+    if (isAdmin === 'loadingPage') return null;
+    if (!isAdmin) return <Redirect to="/" />;
```

Comment:
I think you can consolidate this with the condition on line 75, since you're redirecting to `/` in both cases.